app/src/main/java/com/example/habitsmasher/MainActivity.java

```diff
@@ -58,4 +59,28 @@ public boolean onSupportNavigateUp() {
         return NavigationUI.navigateUp(navController, appBarConfiguration)
                 || super.onSupportNavigateUp();
     }
-}
\ No newline at end of file
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu, adding items to the action bar if present
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if (item.getItemId() == R.id.follow_user_search_button) {
+            openFollowUserDialog();
+            return true;
+        }
+
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void openFollowUserDialog() {
+        FollowUserDialog followUserDialog = new FollowUserDialog();
+        followUserDialog.setCancelable(true);
+        followUserDialog.setTargetFragment(getSupportFragmentManager().getFragments().get(0), 1);
+        followUserDialog.show(getSupportFragmentManager(), "FollowUserDialog");
+    }
+}
```

Comment:
These methods are responsible for inflating the view for the action bar, and adding the selection listener when the user search button is clicked, opening the `FollowUserDialog` box!

This needs to be in the `MainActivity` as this should be available **throughout** the app, not just on one specific page.